integration/spark/src/main/scala/org/apache/spark/sql/CarbonSource.scala

```diff
@@ -281,10 +281,22 @@ object CarbonSource {
       isExternal)
     val updatedFormat = CarbonToSparkAdapter
       .getUpdatedStorageFormat(storageFormat, updatedTableProperties, tableInfo.getTablePath)
```

Comment:
here i can see the handking for `createCatalogTableForCarbonExtension` only, is it handled for `createCatalogTableForCarbonSession`?

Reply:
if its handled, may be in `GeoTableExampleWithCarbonSession.scala` please add some validations just to check if the geo hash column is added in schema for the hive table.

Reply:
added validation, changed the schema ordinal value of geocolumn from -1 to 0, so it is added in to the schema now and handled.